### ISSUE TITLE
wxstart.exe crash after 5-10minutes

### ISSUE THREAD

**Issue by fikriaulia13** [OpenKore/openkore#1980]
Excuse me sir, i wanna ask, why my openkore get crash after i botting 5-10minutes using wxstart.exe, it didnt happen before, the crash just happen this day, thx before sir

**Comment by sctnightcore** [comment]
Do you have pictures?

**Comment by fikriaulia13** [comment]
@sctnightcore here sir... 
![error](https://user-images.githubusercontent.com/26967975/39640188-662d2cb2-4ff4-11e8-8456-79912a2fd407.PNG)

**Comment by sctnightcore** [comment]
try start.exe

**Comment by fikriaulia13** [comment]
@sctnightcore yeah when i use start.exe its never error like that, but i cant use some console command because i didnt know to much about what command can be use at start.exe
hope it can be fixed later the error using wxstart.exe
thx before sir for the time
im appreciate it

**Comment by alisonrag** [comment]
@fikriaulia13 
wxstart is compiled, the last compilation was long time ago, fow now i think no one will can help you with this, you can use.
you can try to use openkore.pl using tha same interface:
https://www.youtube.com/watch?v=3YN-dgWb0t0

**Comment by fikriaulia13** [comment]
@alisonrag wow im trying and successful sir, thank you so much for this information, its new for me, hope using openkore.pl not get any error for this time
thank you very much sir

**Comment by fikriaulia13** [comment]
@alisonrag the error still happening sir... :(
![error2](https://user-images.githubusercontent.com/26967975/39648177-c925d78a-500b-11e8-9155-ee4577c4ae09.PNG)

**Comment by c4c1n6kr3m1** [comment]
try bot on different computer or virtual machine to make sure problem is not in your computer since you said it's worked fine before

**Comment by fikriaulia13** [comment]
@c4c1n6kr3m1 oke sir i'll try it thank you

**Comment by fikriaulia13** [comment]
@c4c1n6kr3m1 i've try to bot on different computer sir,but its still same problem, i think its error on windows 10 :(

**Comment by Fadreus** [comment]
Try with compatibility mode?

**Comment by fikriaulia13** [comment]
@Fadreus compatibilty mode win 7 sir ? yeah i'll try it thankyou sir

**Comment by Pachu1985** [comment]
i have windows 10 and use wxstart almost all the time for all the clients i opened and i never had an error. im pretty sure wxstart is working fine

**Comment by fikriaulia13** [comment]
@Pachu1985 can i get your openkore master sir ? i dont know what happen to my openkore :( still get error even try compatibility win7

**Comment by Fadreus** [comment]
This is outdated but you can try[ this one](https://github.com/OpenKore/openkore/tree/79e394c65fa322cec7b3689d431dd27b990bd62c).
You just need to update the connection files (servers, recvpackets, etc)

I actually still using this version but not wiith OS Win10.
Or you might wanna do some 'surgery' on your Win10.

**Comment by Pachu1985** [comment]
im not sure which version im using, but its an old one and i do just what Fadreus said, only update the connection files
